pkg/kdump/kdump-client.js

```diff
@@ -61,6 +62,19 @@ export class KdumpClient {
 
         // watch the config file
         this.configClient = new ConfigFile("/etc/kdump.conf", true);
+        this._watchConfigChanges();
+
+        this.configClient.wait().then(() => {
+            // if no configuration found, try SUSE version
+            if (this.configClient.settings === null) {
```

Comment:
This is why I suggested to name the file more generically. There is no OS detection for "SUSE" here whatsoever, it's trying the sysconfig format everywhere.

Reply:
I'm fine with more generic name this but it might be confusing for other distros which use `/etc/sysconfig/kdump` but have different format inside.

Reply:
I see; let's leave it as is then, thanks!